{
waypoint_list: [1.5, 0.0, 1.5,
                1.5, -3.5, 1.5,
                -1.5, -3.5, 1.5,
                -1.5, 0.0, 1.5,
                1.5, 0.0, 1.5,
                1.5, 3.5, 1.5,
                -1.5, 3.5, 1.5,
                -1.5, 0.0, 1.5]
}
